lte/gateway/python/scripts/dp_actions_cli.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+import re
+import subprocess
+from magma.common.service import MagmaService
+from lte.protos.mconfig import mconfigs_pb2
+
+def create_parser():
+    """
+    Creates the argparse parser with all the arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description='CLI wrapper around ovs-appctl ofproto/trace.\n'
+                    'To display the Datapath actions of the supplied IMSI',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-i', '--imsi', help='IMSI of the subscriber')
+
+    return parser
+
+def find_ue_ip(imsi: str):
+    """
+    Finds the UE IP address corresponding to the IMSI
+    """
+    cmd = ['mobility_cli.py', 'get_subscriber_table']
+    output = subprocess.check_output(cmd)
+    output_str = str(output, 'utf-8').strip()
+    pattern = 'IMSI.*?' + imsi + '.*?([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})'
+    match = re.search(pattern, output_str)
+    if match:
+        return match.group(1)
+    else:
+        return "NA"
+
+def output_datapath_actions(ue_ip: str):
+    """
+    Returns the Output of Datapath Actions based as per the supplied UE IP
+    """
+    service = MagmaService('pipelined', mconfigs_pb2.PipelineD())
+    if service.mconfig.nat_enabled:
+        in_port = 'local'
+    else:
+        in_port = '2'
```

Comment:
Lets use OVSDB to find port number, `sudo ovsdb-client dump Interface name ofport |grep patch-up`